lib/actors/actor/invocation_scheduler.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Actors.Actor.InvocationScheduler do
+  use GenServer, restart: :transient
+  use Retry
+
+  require Logger
+
+  alias Actors.Registry.ActorRegistry
+  alias Eigr.Functions.Protocol.InvocationRequest
+
+  @impl true
+  def init(_arg) do
+    stored_invocations = ActorRegistry.get_all_invocations()
+
+    Enum.each(stored_invocations, &call_invoke/1)
+
+    {:ok, %{}}
+  end
+
+  @impl true
+  def handle_info({:invoke, request}, state) do
+    decoded_request = InvocationRequest.decode(request)
+
+    retry with: exponential_backoff() |> randomize |> expiry(10_000),
```

Comment:
maybe not necessary or overkill

Reply:
Necessary!

Reply:
Looking more calmly I noticed that you use Actors.invoke further down. In that case you don't need this retry because invoke already implements one. Can remove this retry